torch/_dynamo/eval_frame.py

```diff
@@ -612,6 +682,7 @@ def optimize(
     guard_fail_fn=None,
     disable=False,
     dynamic=None,
+    save_config=True,
```

Comment:
Please add documentation on this API. See: https://github.com/pytorch/pytorch/pull/111074#issuecomment-1759512216

Reply:
Added